### ISSUE TITLE
provisional products based on modis unavilability

### ISSUE THREAD

**Issue by demiurg** [Applied-GeoSolutions/gips#12]
It seems that there are products being generated from atmospherically corrected landsat, that might not be usable for some applications, because of some modis variable being unavailable at the time of processing. This is first a policy issue, and second a gips handling issue.

In terms of handling, I suggest three stages of products, like in PRISM, staging, provisional, stable, to be part of GIPS, somehow.

**Comment by bhbraswell** [comment]
I agree with that. In addition I think that stable products should have a metadata requirement of VERSION. I frequently have to reprocess everything because I didn't record which tiles/ dates used which version.

What is my assignment for this task?
